script/config/31337/42420/targetContractAddresses.json

```diff
@@ -1,20 +1,21 @@
 {
-  "bls": "0xa0Cb889707d426A7A386870A03bc70d1b0697598",
-  "bn256G2": "0xc7183455a4C133Ae270771860664b6B7ec320bB1",
+  "bls": "0x851356ae760d987E095750cCeb3bC6014560891C",
```

Comment:
why some contracts changed and not the other?
1. why implementations changed?
2. why checkpointManager hasn't change - does this mean we don't need it?